src/ratelimit.rs

```diff
@@ -67,18 +67,18 @@ impl RLEntry {
     /// duration is the duration in milliseconds for which the hits are allowed
     fn hit(&mut self, size: u32, duration: u32) -> bool {
         let diff = Instant::now().duration_since(self.epoch);
-        let mut now = u32::try_from(diff.as_millis()).unwrap();
+        let now = u32::try_from(diff.as_millis()).unwrap();
```

Comment:
In rust, shadowing is really used when you "don't care" about the previous value of something, letting the compiler decide what to do with previous value.